docs/next.config.mjs

```diff
@@ -256,6 +256,26 @@ export default withNextra({
         destination: "/store/store-hooks",
         permanent: false,
       },      
+      {
+        source: "/guides/best-practices/dividing-into-systems",
+        destination: "/best-practices/dividing-into-systems",
+        permanent: false,
+      },
+      {
+        source: "/guides/best-practices/system-best-practices",
+        destination: "/best-practices/system-best-practices",
```

Comment:
how about just `/best-practices/systems` instead of duplicating the section title?

Reply:
You're right.